src/BeerTrade/BeerTrade/App.xaml.cs

```diff
@@ -1,16 +1,36 @@
-﻿using System;
+﻿using System.Threading.Tasks;
+using BeerTrade.ViewModels;
+using BeerTrade.Views;
+using Prism;
+using Prism.DryIoc;
+using Prism.Ioc;
 using Xamarin.Forms;
-using Xamarin.Forms.Xaml;
 
 namespace BeerTrade
 {
-    public partial class App : Application
+    public partial class App : PrismApplication
     {
-        public App()
+        public App() : this(null) { }
+
+        public App(IPlatformInitializer platformInitializer) : base(platformInitializer) { }
+
+        protected override async void OnInitialized()
         {
             InitializeComponent();
+            await InitializeNavigation();
+        }
 
-            MainPage = new MainPage();
+        private Task InitializeNavigation()
+            => NavigationService.NavigateAsync("MainPage");
```

Comment:
Maybe ```nameof``` is a better solution here, or not?
```suggestion
            => NavigationService.NavigateAsync(nameof(MainPage));
```

Reply:
In this case it would work as we didn't set a custom alias for the page.